bounded-core/src/main/scala/io/cafienne/bounded/eventmaterializers/AbstractEventMaterializer.scala

```diff
@@ -103,7 +101,7 @@ abstract class AbstractEventMaterializer(
     val source: Source[EventEnvelope, NotUsed] =
       journal
         .eventsByTag(tagName, listenStartOffset)
-        .filter(eventEnvelope => materialzerEventFilter.filter(eventEnvelope.event.asInstanceOf[DomainEvent]))
```

Comment:
materialzerEventFilter was (already) wrongly spelled. Shall we change it in this PR ?

Reply:
I'll change it.